templates/OperationRequestBodySnippet.stencil

```diff
@@ -1,15 +1,15 @@
 // Construct request
 guard let requestBody = try? JSONEncoder().encode( {{ op.request.objectName }} ) else {
-    options?.logger.error("Failed to encode request body as json.")
+    self.options.logger.error("Failed to encode request body as json.")
```

Comment:
Does swiftlint strip out these `self.` usages in the final code? if so, we can remove them here as well for easier maintainability.

Reply:
We need to add 'self.options' to reference the property 'options' of the client class.

Reply:
We need to add 'self' to reference the `options` property of the generated client class.